microgames/hotdog_game/src/hot_dog.gd

```diff
@@ -0,0 +1,15 @@
+extends AnimatedSprite
+
+
+var FALL_SPEED = 16
+var is_falling = false
+
+func _ready():
+	position.x = rand_range(200, 700)
```

Comment:
instead of hardcoding these numbers you could add `Position2D` nodes to mark the boundaries instead

or maybe do some fancy math with the viewport size or whatever? it would be nice if there weren't node dependencies so this script was easy to use elsewhere....could maybe export the boundary values/nodepaths

idk haha i don't have an Ideal Solution off-hand